src/query/sql/src/executor/physical_plans/physical_window.rs

```diff
@@ -55,6 +55,7 @@ pub struct Window {
     pub partition_by: Vec<IndexType>,
     pub order_by: Vec<SortDesc>,
     pub window_frame: WindowFuncFrame,
+    pub limit: Option<usize>,
```

Comment:
We can add `limit:xxx` to `window_to_format_tree` in `src/query/sql/src/executor/format.rs` to display limit in explain.